e2e/framework/framework.go

```diff
@@ -155,11 +155,18 @@ func InitializeCluster() {
 	}
 
 	var application *loc.Locator
-	if mode == wizardMode {
-		Cluster, application, err = infra.NewWizard(config, provisioner, installerNode)
-		TestContext.Application.Locator = application
-	} else {
-		Cluster, err = infra.New(config, TestContext.OpsCenterURL, provisioner)
+	switch {
+	case mode == provisionMode:
+		return
```

Comment:
If you exit early here, no cluster state is saved into a state file which means robotest binary will not be able to shut down the infrastructure. I think you don't need this as extra mode - just disabling the test should work w/o further modifications necessary.